kubernetes-dashboard-api.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: kubernetes-dashboard-api
+  version: 1.10.1
+  epoch: 0
+  description: Go module handling authentication to the Kubernetes API
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/dashboard
+      tag: api/v${{package.version}}
+      expected-commit: 8c15a76aec0489f63ab841e4aaf09391d2e68912
+
+  - uses: go/build
+    with:
+      packages: .
+      modroot: ./modules/api
+      output: dashboard-api
+
+  - uses: strip
```

Comment:
we can omit this given go/build pipeline already strips the binary. ref: https://github.com/chainguard-dev/melange/blob/7af591847a044b2eb12246dcabf1f4e5346cd9ea/pkg/build/pipelines/go/build.yaml#L58-L62
```suggestion

```